File: app/src/main/java/ru/alexsuvorov/paistewiki/Fragment/ArtistsFragment.java
package ru.alexsuvorov.paistewiki.Fragment;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import ru.alexsuvorov.paistewiki.R;

public class ArtistsFragment extends Fragment {

    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.artists_fragment, container, false);
    }

}

File: feature/artist_list/impl/src/main/java/ru/alexsuvorov/paistewiki/feature/artist_list/ArtistsFragment.kt
package ru.alexsuvorov.paistewiki.feature.artist_list

import android.content.Context
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import ru.alexsuvorov.paistewiki.tools.AppPreferences

class ArtistsFragment : Fragment() {

    private var appPreferences: AppPreferences? = null

    override fun onAttach(context: Context) {
        super.onAttach(context)
        appPreferences = AppPreferences(requireContext())
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_artists, container, false)
    }
}